<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbChangeRegular">
        <property name="text">
         <string>Regular calculator</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbChangeMatrix">
        <property name="text">
         <string>Matrix calculator</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbChangePlotting">
        <property name="text">
         <string>Plotting calculator</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="p0Regular">
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Regular</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="leParser">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>1198</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pbParser">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>110</y>
            <width>1198</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Izracunaj</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="p1Matrix">
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>610</x>
            <y>130</y>
            <width>571</width>
            <height>131</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="1">
            <widget class="QPushButton" name="pbMatrixOne">
             <property name="text">
              <string>One</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pbMatrixInverse">
             <property name="text">
              <string>Inverse</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pbMatrixTranspose">
             <property name="text">
              <string>Transpose</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pbMatrixDiag">
             <property name="text">
              <string>Diag</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>130</y>
            <width>551</width>
            <height>131</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QPushButton" name="pbMatrixSubtract">
             <property name="text">
              <string>Subtract</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pbMatrixAdd">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pbMatrixMultiply">
             <property name="text">
              <string>Multiply</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pbMatrixDivide">
             <property name="text">
              <string>Divide</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="formLayoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>10</y>
            <width>221</width>
            <height>108</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Dimension 1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="leMatrixDim11"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="leMatrixDim12"/>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLineEdit" name="leMatrixData1"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Dimension 2</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QPushButton" name="pbMatrixLoad1">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="formLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>610</x>
            <y>10</y>
            <width>221</width>
            <height>108</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Dimension 1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="leMatrixDim21"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="leMatrixDim22"/>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLineEdit" name="leMatrixData2"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Dimension 2</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QPushButton" name="pbMatrixLoad2">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="p2Plotting">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Plotting</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Resenje</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="tbParser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
